src/w3d/math/gamemath.h

```diff
@@ -533,7 +533,7 @@ inline int Lrintf(float val)
 inline bool Is_Nan(float val)
 {
 #ifdef BUILD_WITH_GAMEMATH
-    return gm_isnanf(val);
+    return static_cast<bool>(gm_isnanf(val));
```

Comment:
If `gm_isnanf(val)` returns int, then I would write it as `return gm_isnanf(val) != 0;`